pydantic/fields.py

```diff
@@ -470,6 +478,8 @@ def Field(
         json_schema_extra=json_schema_extra,
         strict=strict,
         validate_default=validate_default,
+        validation_alias=validation_alias or alias,
+        serialization_alias=serialization_alias or alias,
```

Comment:
i think we should only set `serialization_alias` to `alias` if `alias` is a string.

Reply:
[alias type is str|None](https://github.com/pydantic/pydantic/blob/1896500fb6bc371270b94f343390e2121dc0e193/pydantic/fields.py#L318), How can it be possible to have non string alias?

Reply:
done